### DISCUSSION TITLE
Welcome to Muncoordinated's discussion space!

### DISCUSSION THREAD

**Discussion by MaxwellBo** [MaxwellBo/Muncoordinated-2#176] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alexande-rJF** [comment]
I'm Chairing a committee which is a Historical Crisis, I was wondering if there was a way to add custom speakers, I.E. the Kingdom of Sardinia, Kingdom of Prussia, Ottoman Empire, etc.

**Reply by MaxwellBo** [reply]
You can create custom entities in the Setup tab:

<img width="314" alt="image" src="https://github.com/user-attachments/assets/5abd6ef7-d6de-40dc-b89e-c7b0a2ca8594" />

**Reply by Alexande-rJF** [reply]
Thank you so much for your help!

**Comment by BAntu10** [comment]
I am going to be chairing an muns soon. I wanted to add a few motions like cotw, tour de table, appeal chairs decision and in voting the future of voting with rights, so how do you suppose I can do that is there any feature or do I have to edit the code

**Reply by MaxwellBo** [reply]
There is sadly not any way to add custom motions and it's unlikely Muncoordinated will ever support that.

**Reply by BAntu10** [reply]
Hey so I edited the code and was able to add 10 more motions and add a voting feature for motions instead of straight out opening them. I also managed to differentiate between substantive and procedural vote. Also added 2 more vote types.
Yes with rights 
No with rights

What am I supposed to do with it I am hosting it on my domain but do you want to make it the official version??

**Reply by MaxwellBo** [reply]
You will need to fork the repository and then open a Pull Request as per the instructions here: [Fork a repository - Propose changes to someone else's project](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#propose-changes-to-someone-elses-project)

**Comment by BAntu10** [comment]
Hey so I edited the code and was able to do it thanks for replying though...
How shall I post the code though...
